lumicks/pylake/confocal.py

```diff
@@ -133,6 +133,20 @@ def _ordered_axes(self):
         """Returns axis indices in spatial order"""
         return sorted(self.json["scan volume"]["scan axes"], key=lambda x: x["axis"])
 
+    def _to_spatial(self, data):
+        raise NotImplementedError
```

Comment:
Add a docstring here and for `_shape` so we know what this needs to do. The implementations can differ but the intent is the same.